### PR TITLE
fix: #CRRE-521 fix decredit bookseller

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
When we send an order to the bookseller, the price corresponds to the unit price. it must therefore be multiplied by the quantity.

## Checklist tests
Changing the status of an order, we are decredited/credited. 
Send an order refused to the bookseller, we are well discredited.

## Issue ticket number and link
[CRRE-521](https://jira.support-ent.fr/browse/CRRE-521)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/ts/model/OrderRegion.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export class OrderRegion implements Order  {
     toJson():any {
         return {
             amount: this.amount,
-            price: this.price,
+            price: this.price / this.amount,
```
Comment (sjollois) [inline src/main/resources/public/ts/model/OrderRegion.ts:56]: C'est dangereux ça...

Comment (sjollois) [reply src/main/resources/public/ts/model/OrderRegion.ts:56]: ça ne fais pas une division exacte, ça te sort un chiffre entier il me semble

**Review by sjollois — CHANGES_REQUESTED** [review]

File: src/main/resources/public/ts/model/OrderRegion.ts (hunk 1 above)
Comment (sjollois) [reply src/main/resources/public/ts/model/OrderRegion.ts:56]: et c'est déjà le prix unitaire de l'article, il me semble!

Comment (BastienBenard) [reply src/main/resources/public/ts/model/OrderRegion.ts:56]:
```suggestion
            price: this.amount ? this.price / 1.0 / this.amount : 0.0
```

**Review by sjollois — APPROVED** [review]

**Pull request merged**